tests/resolvers/boolean.test.ts

```diff
@@ -2,42 +2,42 @@ import { Identifiers, Resolvers } from '../../src';
 
 describe('Boolean resolver tests', () => {
 	test('GIVEN a truthy value THEN returns true', () => {
-		expect(Resolvers.resolveBoolean('true').success).toBe(true);
-		expect(Resolvers.resolveBoolean('true').error).toBeUndefined();
+		expect(Resolvers.resolveBoolean('true').isOk()).toBe(true);
+		expect(Resolvers.resolveBoolean('true').unwrapErr).toThrowError();
```

Comment:
You don't need to test `Result`, you need to test values.
```suggestion
		expect(Resolvers.resolveBoolean('true')).toEqual(Result.ok(true));
```

Same applies everywhere else, unit tests that can throw are faulty and hard to debug.